pbft/test/pbft_proto_test.cpp

```diff
@@ -66,7 +67,7 @@ namespace bzn
     void
     pbft_proto_test::send_preprepare(uint64_t sequence, const bzn_envelope& request)
     {
-        // after preprepare is sent, SUT will send out prepares to all nodes
+        // after preprepares is sent, SUT will send out prepares to all nodes
```

Comment:
?

Reply:
Removed the comment.

Reply:
also found another misspelling of their...